plugins/codemirror/dialogs/codemirrordialog.js

```diff
@@ -236,6 +236,10 @@ CKEDITOR.dialog.add('codemirrordialog', function (editor) {
 
 			if (newData !== oldData) {
 				editor.setData(newData);
+
+				if (editor.balloonToolbars) {
+					editor.balloonToolbars.hide();
+				}
```

Comment:
Thanks @diegonvs

Reply:
onLoad will not work because it is called just one time when the plugin is loaded. When moving to onShow it looks like a Tarsila do Amaral paint. Thanks Julien